.circleci/config.yml

```diff
@@ -45,37 +79,26 @@ jobs:
 
     steps:
     - checkout
-
-    # Download and cache dependencies
-    - restore_cache:
-        keys:
-        - v1-dependencies-{{ checksum "package.json" }}-{{checksum "package-lock.json"}}
-
-    # set node version to 8.9.0. This is more complicated than it needs to be
-    # because each 'run' script runs in its own shell. If we switch to docker we can remove this
-    # see https://discuss.circleci.com/t/how-to-change-node-version-in-circleci-2-0/17455/4
-    - run:
-        name: Install node@8.9.0
-        command: |
-          set +e
-          curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.5/install.sh | bash
-          export NVM_DIR="/opt/circleci/.nvm"
-          [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
-          nvm install v8.9.0
-          nvm alias default v8.9.0
-          # Each step uses the same `$BASH_ENV`, so need to modify it
-          echo 'export NVM_DIR="/opt/circleci/.nvm"' >> $BASH_ENV
-          echo "[ -s \"$NVM_DIR/nvm.sh\" ] && . \"$NVM_DIR/nvm.sh\"" >> $BASH_ENV
-
+    - *cache_restore
+    - *install_node
     - run: npm install
-
-    - save_cache:
-        paths:
-        - node_modules
-        key: v1-dependencies-{{ checksum "package.json" }}-{{checksum "package-lock.json"}}
-
+    - *cache_save
     - run:
         name: Pull latest docker images
         command: |
           docker-compose -f test/integration/docker-geth/docker-compose.yml pull
-    - run: npm run integration-test-geth
\ No newline at end of file
+    - run: npm run integration-test-geth
+
+  build-marketplace-tx-server-latest:
+    docker:
+      - image: circleci/node:8.9
+
+    steps:
+      - run: curl -d "build_parameters[CIRCLE_JOB]=build-docker-latest" "https://circleci.com/api/v1.1/project/github/civicteam/civic_js_node_server/tree/dev?circle-token=${CIRCLE_TOKEN}"
+
+  build-marketplace-tx-server-test:
+    docker:
+      - image: circleci/node:8.9
+
+    steps:
```

Comment:
Is dev the correct branch here? BTW I believe workflow triggers are now possible on circleci (at least somewhat) https://circleci.com/docs/api/v1-reference/ (See /build)

https://circleci.com/docs/api/v1-reference/#new-project-build

Might be worth experimenting with this as this is a much better option than triggering jobs directly.

Reply:
Right. It should point to master branch here. Thanks.